process-engine/bpmn-studio
# ✨ Add EndEvent name and ID to CallActivity result

**Changes:**

1. Update `ProcessEngineRuntime` to include an EndEvent's name and ID with each CallActivity result.
2. Remove unused `process_model.sqlite` file.


**Issues:**

Part of https://github.com/process-engine/process_engine_runtime/issues/280

PR: #1393

## How can others test the changes? 

- Run a Process that uses a CallActivity
- After it finishes, go to the `Inspect` view and inspect the ProcessTokens for the ProcessInstance
- See that the onExit token for the CallActivity now contains three properties:
    - result: The CallActivity's final result
    - endEventName: The name of the EndEvent that finished the CallActivity's SubProcess
    - endEventId: The ID of the EndEvent that finished the CallActivity's SubProcess

![bildschirmfoto 2019-03-05 um 12 55 48](https://user-images.githubusercontent.com/15343316/53803776-06737000-3f46-11e9-9f39-33be45403712.png)

## PR-Checklist

Please check the boxes in this list after submitting your PR:
